init.lua

```diff
@@ -173,8 +181,10 @@ local plug_require = function(plug, args)
 	if not file_exists(plug.path) then
 		return
 	end
-	local plugin_name = plug.name .. '/' .. plug.file
-	local plugin = require(plugin_name)
+	if not plug.theme then
+		local plugin_name = plug.name .. '/' .. plug.file
+		local plugin = require(plugin_name)
```

Comment:
you are making `plugin` undefined here for usage with M.plugins

Reply:
Should the theme be added as a plugin?

Reply:
No, but in the line under i set plugin to M.plugins, so you need to include that.

Reply:
Shifted in the last commit the lower if inside the other if

Reply:
How should the ```if``` statements be combined?
Should a theme also have an alias?

Reply:
> And please, rebase your commits into a single commit when finished.

I will rebase after all your complains are fixed

Reply:
My mistake. You can't combine the if´s, just indent the alias check, like it looks like this:

```lua
	if not plug.theme then
		local plugin_name = plug.name .. '/' .. plug.file
		local plugin = require(plugin_name)
		if plug.alias then
			M.plugins[plug.alias] = plugin
		end
	end
```